packages/admin/admin/src/mui/menu/Item.tsx

```diff
@@ -1,144 +1,185 @@
+                }
+                &:hover {
+                    background-color: ${theme.palette.primary.dark};
+                }
+                .MuiListItemText-primary {
+                    font-weight: ${theme.typography.fontWeightBold};
+                }
+            }
+
+            .MuiListItemText-root {
+                margin: 0;
+            }
+
+            .MuiListItemText-primary {
+                font-weight: ${theme.typography.fontWeightRegular};
+                font-size: 14px;
+                line-height: 20px;
+            }
+        `};
 
-export interface MenuItemProps extends MenuLevel {
+        ${ownerState.secondaryText}
+
+        ${ownerState.secondaryText}
+
+        ${ownerState.secondaryAction &&
+        css`
+            padding-right: 18px;
+        `}
+    `,
+);
+
+export interface MenuItemProps extends ThemedComponentBaseProps<{ root: typeof ListItem }>, MenuLevel {
     primary: React.ReactNode;
     secondary?: React.ReactNode;
     icon?: React.ReactElement;
     secondaryAction?: React.ReactNode;
+    secondaryText?: boolean;
 }
 
 type MuiListItemProps = Pick<ListItemProps, Exclude<keyof ListItemProps, "innerRef" | "button">> & { component?: React.ElementType };
 
-const Item: React.FC<WithStyles<typeof styles> & MenuItemProps & MuiListItemProps> = ({
-    classes,
-    primary,
-    secondary,
-    icon,
-    level = 1,
-    secondaryAction,
-    ...otherProps
-}) => {
-    const context = React.useContext(MenuContext);
-    if (!context) throw new Error("Could not find context for menu");
-    if (level > 2) throw new Error("Maximum nesting level of 2 exceeded.");
+export function MenuItem(inProps: MenuItemProps & MuiListItemProps) {
+    const {
+        primary,
+        secondary,
+        icon,
+        level = 1,
+        secondaryAction,
+        secondaryText,
+
+        slotProps,
+        ...otherProps
+    } = useThemeProps({
+        props: inProps,
+        name: "CometAdminMenuItem",
+    });
+
+    const ownerState: OwnerState = {
+        level,
+        icon,
+        secondaryText,
+        secondaryAction,
+    };
 
     const hasIcon = !!icon;
 
-    const listItemClasses = [classes.root];
-    if (level === 1) listItemClasses.push(classes.level1);
-    if (level === 2) listItemClasses.push(classes.level2);
-    if (hasIcon) listItemClasses.push(classes.hasIcon);
-    if (secondary) listItemClasses.push(classes.hasSecondaryText);
-    if (secondaryAction) listItemClasses.push(classes.hasSecondaryAction);
+    const context = React.useContext(MenuContext);
+    if (!context) throw new Error("Could not find context for menu");
+    if (level > 2) throw new Error("Maximum nesting level of 2 exceeded.");
 
     return (
-        <ListItem component="div" button classes={{ root: listItemClasses.join(" ") }} {...otherProps}>
+        <Root component="div" button {...slotProps?.root} ownerState={ownerState} {...otherProps}>
```

Comment:
@jamesricky I still need a solution for the button prop as its usage is deprecated for the ListItem component (and lint fails). The MUI documentation recommends using the ListItemButton instead (https://mui.com/material-ui/api/list-item/#list-item-prop-button). Should I refactor or is this a task for another PR...?

Reply:
I don't understand why using `styled(ListItem)` breaks the type of this prop 🤔

But yes, since it's deprecated and the props of this component generally seem a bit confusing maybe it's a good time to change it to use `ListItemButton`.

Then I think we should simplify the props, since this is already a breaking change. 
The props should extend `ThemedComponentBaseProps`, `MenuLevel` and `ListItemButtonProps` and the `MuiListItemProps` should be removed.

Reply:
@jamesricky I am not sure: Do I need to write a changeset for changing the ListItem to ListItemButton? If yes, can you please help me, with the information I need to provide in the changeset?

Reply:
Thank you!